src/orquestra/sdk/_base/_driver/_client.py

```diff
@@ -590,7 +611,10 @@ def get_workflow_run_result(
         """
 
         resp = self._get(
-            API_ACTIONS["get_workflow_run_result"].format(result_id), query_params=None
+            self._uri_provider.uri_for(
+                "get_workflow_run_result", parameters=(result_id,)
+            ),
+            query_params=None,
```

Comment:
**non-blocking,question**: does the addition of `query_params=None` change anything here?

Reply:
No, but its required argument. You can see it was there before, just in another line

Reply:
oooh, right.